synth.metadata

```diff
@@ -42,5 +42,299 @@
         "generator": "bazel"
       }
     }
+  ],
+  "generatedFiles": [
```

Comment:
@tbpg @lesv is this expected to be added to the synth metadata?

Reply:
@chingor13 PTAL
I don't recognize any of it.  It maybe for Jeff's latest changes.